include/libtorrent/kademlia/dht_tracker.hpp

```diff
@@ -69,14 +71,18 @@ namespace libtorrent { namespace dht {
 			, counters& cnt
 			, dht_storage_interface& storage
 			, dht_state state);
-		virtual ~dht_tracker();
+
+		dht_tracker(dht_tracker const&) = delete;
```

Comment:
Now I remember why I did this. It's a workaround for a bug in msvc 14.0 which is fixed in msvc 14.1. I've re-added the line with an appropriate `#if` condition and comment.